pom.xml

```diff
@@ -37,8 +37,9 @@
 
     <properties>
         <!-- Change the nd4j.backend property to nd4j-cuda-9.0-platform nd4j-cuda-9.2-platform or nd4j-cuda-10.0-platform to use CUDA GPUs -->
-        <nd4j.backend>nd4j-native-platform</nd4j.backend>
+        <nd4j.backend>nd4j-cuda-10.0-platform</nd4j.backend>
```

Comment:
Right, dl4j overwrites nd4j backend. 

Hence if i have cuda and cudnn installed, 
and for now in the [deeplearning4j-cuda-*.*](https://github.com/skymindglobal/TrainingLabs/blob/master/dl4j-labs/pom.xml#L103-L108) in dl4j-labs of master branch is enabled by default, 
when i pulled down the repo, it default runs on cuda backend.

so for this PR main thing is i want to comment out [deeplearning4j-cuda-10.0](https://github.com/skymindglobal/TrainingLabs/blob/master/dl4j-labs/pom.xml#L103-L108) by default. that is better since people might have different version as well..

for [cuDNN error](https://gist.github.com/willardsm/1e024d18f0630bea9517098bf114c4ec#file-cudnn-error-L12-L36) you mentioned, 
yeah it's common

CUDA and cudnn can be enable individually, 

mean we can run cuda backend but not having cudnn , then we will see that warning
alternatively, we can have cuda + cudnn, then it by right should be faster than before

by the way cudnn only accelerate certain layers, such as CNN and LSTM, 
if you run feedforward, cudnn is not utilized so the warning would not pop up either way.

Reply:
i agree with you, we should comment out deeplearning4j-cuda-10.0 by default.

since we comment out the deeplearning4j-cuda-10.0 by default, should we also change the nd4j backend to native `<nd4j.backend>nd4j-native-platform</nd4j.backend>`? 
because in current commit the nd4j backend is `<nd4j.backend>nd4j-cuda-10.0-platform</nd4j.backend>`. Not sure that was intended or a mistake?

In overall, the PR is good to merge